OCIContainer/OCIContainer.json

```diff
@@ -39,6 +39,11 @@
             "properties": {
                 "success": {
                     "$ref": "#/definitions/success"
+                },
+                "error":{
+                        "summary": "An error message in case of a failure",
```

Comment:
For entire json API definition please use [schema](https://github.com/rdkcentral/Thunder/blob/master/Tools/JsonGenerator/schemas/interface.schema.json) defined in Thunder.

```json         
"errors": {
  "description": "Definition of possible error responses.",
  "type": "array",
  "items": {
    "description": "Definition of an error response.",
    "type": "object",
    "properties": {
      "description": {
        "description": "Description of the failure.",
        "type": "string"
      },
      "code": {
        "description": "Error code.",
        "type": "number"
      },
      "message": {
        "description": "Error message.",
        "type": "string"
      }
    },
    "required": [
      "code",
      "message"
    ]
  }
}
```
According to the schema errors (not error) param is an array of items with mandatory error code and message fields so this should be changed to:

```json
"errors": [
    {
        "description": "Internal Dobby Error",
        "$ref": "#/common/errors/internal_dobby_error"
    }
]
```
where "$ref": `#/common/errors/internal_dobby_error` can be define in common section of OCIContainer.json (before definitions) eg.

```json
"common": {
    "errors": {
        "internal_dobby_error": {
            "code": 1,
            "message": "ERROR_DOBBY_INTERNAL"
        },
        ...
    }
}
```
Please see example in Packager:
https://github.com/rdkcentral/rdkservices/blob/92b0ed86dda3d2ce845395f5ab61d1ef09bb3a1e/Packager/Packager.json

Reply:
Hi @piotr-serafin-epam. As we discussed current proposition is compatible with other plugins error handling as well as with JSON schema which gives possibility to add optional items (eg error in result object).

Reply:
Ok sounds good, let's use result object instead to keep backward compatibility with existing OCIContainer clients. Please proceed with extending error reporting.